config/eslint-config-ironfish/index.js

```diff
@@ -105,5 +105,12 @@ module.exports = {
         groups: [['\\u0000', '^@?\\w', '^', '\\.']],
       },
     ],
+
+    "jest/expect-expect": [
```

Comment:
Could you put this in with the other `jest` rules above and add a comment for the reasoning for it?

Reply:
Moved to be with the other `jest` rules and added a comment for the reasoning